### PR TITLE
image-width shortcode. This shortcode adds an image with a restricted…

### PR DESCRIPTION
**Pull request by micronAMZN**
… width and alt text. Need this for .svgs that scale up infinitely.

Signed-off-by: Mike Cronin <mikecro@amazon.com>

## Change summary

Adding a shortcode that will add an image with alt text that is restricted to a specified width. Need this to keep svgs from scaling to ridiculous sizes. Can be used to scale screenshots that are very large as well, though the final image quality will be affected by the browser.

### Submission Checklist:

* [ ] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [ ] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [ ] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [ ] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/contributing/to-docs/style-guide/media.md [hunk 1]
```diff
@@ -18,6 +22,31 @@ Image output:
 
 ![O3DE Logo](/img/logos/O3DE-Logo-with-WordMark-Black-Mono.svg "The O3DE logo")
 
+## Adding images with the `image-width` shortcode
+
+The `image-width` shortcode adds an image with alternate text and restricts the image's width. The `image-width` shortcode can ensure image sizes are consistent within a topic, and that large images don't scale overly large in wide browser windows. Use this method when adding `.svg` diagrams, very large images, and in topics with multiple images where consistent image widths are desirable.
+
+`image-width` takes three parameters in order:
+
+1. image name
```
Comment (willihay) [inline content/docs/contributing/to-docs/style-guide/media.md:31]:
```suggestion
1. image link
```

File: content/docs/contributing/to-docs/style-guide/media.md [hunk 2]
```diff
@@ -18,6 +22,31 @@ Image output:
 
 ![O3DE Logo](/img/logos/O3DE-Logo-with-WordMark-Black-Mono.svg "The O3DE logo")
 
+## Adding images with the `image-width` shortcode
+
+The `image-width` shortcode adds an image with alternate text and restricts the image's width. The `image-width` shortcode can ensure image sizes are consistent within a topic, and that large images don't scale overly large in wide browser windows. Use this method when adding `.svg` diagrams, very large images, and in topics with multiple images where consistent image widths are desirable.
+
+`image-width` takes three parameters in order:
```
Comment (willihay) [inline content/docs/contributing/to-docs/style-guide/media.md:29]:
```suggestion
`image-width` takes three double-quoted parameters in order:
```

File: content/smoketest.md [hunk 3]
```diff
@@ -419,5 +419,22 @@ You can add inline O3DE GUI icons with the `icon` shortcode. Icon `.svg` files a
 | visibility-on.svg | {{< icon "visibility-on.svg" >}} |
 | world.svg | {{< icon "world.svg" >}} |
 
+## Images and image width
 
+The `image-width` shortcode adds an image with alternate text and restricts the image's width. The `image-width` shortcode can ensure image sizes are consistent within a topic, and that large images and `.svg` diagrams don't scale overly large in wide browser windows.
 
+`image-width` takes three parameters in order:
```
Comment (willihay) [inline content/smoketest.md:426]:
```suggestion
`image-width` takes three double-quoted parameters in order:
```

File: content/smoketest.md [hunk 4]
```diff
@@ -419,5 +419,22 @@ You can add inline O3DE GUI icons with the `icon` shortcode. Icon `.svg` files a
...
+
+1. image name
```
Comment (willihay) [inline content/smoketest.md:428]:
```suggestion
1. image link
```

**Review by willihay — CHANGES_REQUESTED** [review]

File: content/docs/contributing/to-docs/style-guide/media.md [hunk 5]
```diff
@@ -18,6 +22,31 @@ Image output:
...
+```
+
+`image-width` example output:
+
+{{< image-width "/images/welcome-guide/wg-welcome-page-color.png" "700" "The O3DE Welcome Guide splash image." >}}
+
+
+## Alternate text
+
+Alternate text for images is very important. It provides a description of the image if the image is not loaded. The alternate text is also used in accessibility scenarios and in search engine optimization (SEO). Ensure that the alternate text provided for images clearly describes the contents of the image.
```
Comment (chanmosq) [inline content/docs/contributing/to-docs/style-guide/media.md:48]:
```suggestion
Alternate text for images is very important. It provides a description of the image if the image is not loaded. The alternate text is also used for accessibility and in search engine optimization (SEO). Ensure that the alternate text provided for images clearly describes the contents of the image. 
```

**Review by chanmosq — APPROVED** [review]
Minor, non-blocking suggestions.

**Review by willihay — APPROVED** [review]

**Pull request merged**